### PR TITLE
Add stormssh completion (bash)

### PR DESCRIPTION
**Pull request by vigo**
Bash completion for [stormssh](https://github.com/emre/storm). Details can be found [here](https://github.com/vigo/stormssh-completion)

### CODE REVIEW

**Comment by vigo** [comment]
tests did not run? how come? anyone help me out?

File: stormssh-completion.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class StormSshCompletion < Formula
```
Comment (bfontaine) [inline stormssh-completion.rb]: This should match the filename. So either change this to `StormsshCompletion` or change the filename to `storm-ssh-completion.rb`.

Comment (vigo) [inline stormssh-completion.rb]: thanx

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @vigo; we appreciate it! :tada:

**Pull request closed without merging**